packages/flutter_localizations/lib/src/l10n/cupertino_kn.arb

```diff
@@ -1,22 +1,23 @@
 {
-  "datePickerHourSemanticsLabelOne": "\u0024\u0068\u006f\u0075\u0072\u0020\u0c97\u0c82\u0c9f\u0cc6",
-  "datePickerHourSemanticsLabelOther": "\u0024\u0068\u006f\u0075\u0072\u0020\u0c97\u0c82\u0c9f\u0cc6",
-  "datePickerMinuteSemanticsLabelOne": "\u0031\u0020\u0ca8\u0cbf\u0cae\u0cbf\u0cb7",
-  "datePickerMinuteSemanticsLabelOther": "\u0024\u006d\u0069\u006e\u0075\u0074\u0065\u0020\u0ca8\u0cbf\u0cae\u0cbf\u0cb7\u0c97\u0cb3\u0cc1",
-  "datePickerDateOrder": "\u0064\u006d\u0079",
-  "datePickerDateTimeOrder": "\u0064\u0061\u0074\u0065\u005f\u0074\u0069\u006d\u0065\u005f\u0064\u0061\u0079\u0050\u0065\u0072\u0069\u006f\u0064",
-  "anteMeridiemAbbreviation": "\u0cac\u0cc6\u0cb3\u0cbf\u0c97\u0ccd\u0c97\u0cc6",
-  "postMeridiemAbbreviation": "\u0cb8\u0c82\u0c9c\u0cc6",
-  "todayLabel": "\u0c87\u0c82\u0ca6\u0cc1",
-  "alertDialogLabel": "\u0c8e\u0c9a\u0ccd\u0c9a\u0cb0\u0cbf\u0c95\u0cc6",
-  "timerPickerHourLabelOne": "\u0c97\u0c82\u0c9f\u0cc6",
-  "timerPickerHourLabelOther": "\u0c97\u0c82\u0c9f\u0cc6\u0c97\u0cb3\u0cc1",
-  "timerPickerMinuteLabelOne": "\u0ca8\u0cbf\u0cae\u0cbf\u002e",
-  "timerPickerMinuteLabelOther": "\u0ca8\u0cbf\u0cae\u0cbf\u002e",
-  "timerPickerSecondLabelOne": "\u0cb8\u0cc6\u002e",
-  "timerPickerSecondLabelOther": "\u0cb8\u0cc6\u002e",
-  "cutButtonLabel": "\u0c95\u0ca4\u0ccd\u0ca4\u0cb0\u0cbf\u0cb8\u0cbf",
-  "copyButtonLabel": "\u0ca8\u0c95\u0cb2\u0cbf\u0cb8\u0cbf",
-  "pasteButtonLabel": "\u0c85\u0c82\u0c9f\u0cbf\u0cb8\u0cbf",
-  "selectAllButtonLabel": "\u0c8e\u0cb2\u0ccd\u0cb2\u0cb5\u0ca8\u0ccd\u0ca8\u0cc2\u0020\u0c86\u0caf\u0ccd\u0c95\u0cc6\u0cae\u0cbe\u0ca1\u0cbf"
+  "datePickerHourSemanticsLabelOne": "$hour ಗಂಟೆ",
```

Comment:
This requires running `flutter/dev/tools/localization/bin/encode_kn_arb_files.dart`. There's a bug with emacs that causes the editor to crash when it detects text in Kannada script. 

I need to still make this part of the `gen_localizations` and `gen_date_localizations` scripts.

Reply:
Awesome! Thank you! Updates on the way.